src/commands/schema/regenerate.ts

```diff
@@ -0,0 +1,57 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { apiDocsURL } from '../../lib/command/api-command.js'
+import {
+	apiOrganizationCommand,
+	apiOrganizationCommandBuilder,
+	type APIOrganizationCommandFlags,
+} from '../../lib/command/api-organization-command.js'
+import { outputItem, outputItemBuilder, OutputItemFlags } from '../../lib/command/output-item.js'
+import { chooseSchemaApp } from '../../lib/command/util/schema-util.js'
+
+
+export type CommandArgs =
+	& APIOrganizationCommandFlags
+	& OutputItemFlags
+	& {
+		id?: string
+	}
+
+const command = 'schema:regenerate [id]'
+
+const describe = 'regenerate the client id and secret of the Schema App link'
+
+const builder = (yargs: Argv): Argv<CommandArgs> =>
+	outputItemBuilder(apiOrganizationCommandBuilder(yargs))
+		.positional('id', {
+			describe: 'Schema App link id',
+			type: 'string',
+		})
+		.example([
+			['$0 schema:regenerate', 'prompt for Schema App link and regenerate the client id and secret for it'],
+			[
+				'$0 schema:regenerate 392bcb11-e251-44f3-b58b-17f93015f3aa',
+				'regenerate the client id and secret for for the specified Schema App link',
+			],
+		])
+		.epilog('NOTE: The previous values will be invalidated, which may affect existing installations.\n\n' +
+			apiDocsURL('generateStOauthCredentials'))
+
+const handler = async (argv: ArgumentsCamelCase<CommandArgs>): Promise<void> => {
+	const command = await apiOrganizationCommand(argv)
+
+	const id = await chooseSchemaApp(
+		command,
+		argv.id,
+		{ promptMessage: 'Select a schema app to regenerate its client id and secret.' },
+	)
```

Comment:
Can/should we repeat the warning about the old values being invalidated (and thus potentially breaking any existing installations) here as well?  Not everyone takes the time to read documentation.

Reply:
That's a good idea but putting it here would only help those who use it in Q&A mode. I think it it would be better prompt for confirmation with the warning. I wrote an issue for it though since that's a bit more than a refactor (even though it is pretty simple).